packages/experimental/Shadow/src/Shadow.types.ts

```diff
@@ -1,7 +1,23 @@
 import { ViewProps } from 'react-native';
 
-export type ShadowDepth = '2' | '4' | '8' | '16' | '28' | '64';
+export const shadowName = 'Shadow';
+
+export type ShadowDepth =
```

Comment:
You could also have two props. one for `ShadowDepth` and one for `isBrand`

Reply:
Going to leave it for now, but thank you for the suggestion!